cmd/program/model.go

```diff
@@ -372,6 +386,47 @@ func (m *Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			return m, nil
 		}
 
+		if m.Mode == ModeSeed {
+			if m.SnapshotDatabase {
+				m.Status = StatusSnapshotDatabase
+				return m, SnapshotDatabase(m.ProjectDir, m.Schema, m.Database)
+			}
+
+		}
+
+		// If seed data flag is set, run seed data
+		if m.SeedData {
+			// If seed data directory exists, run seed data
+			seedDir := filepath.Join(m.ProjectDir, "seed")
+			if _, err := os.Stat(seedDir); !os.IsNotExist(err) {
```

Comment:
```suggestion
			if _, err := os.Stat(seedDir); err == nil {
```

I think we need to attempt to seed only if there aren't any errors